test_utilities.py

```
@@ -0,0 +1,438 @@
+"""
+Test Utilities
+
+This module provides centralized test utilities to eliminate code duplication
+in mock object creation and test setup patterns across test files.
+"""
+
+import discord
+from unittest.mock import MagicMock, AsyncMock, Mock
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)